File: moregorenine/baekjoon/step3_for/q5_2741/Main.java
package baekjoon.step3_for.q5_2741;

import java.io.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        try {
            int count = Integer.parseInt(br.readLine());
            printUntilNumber(count);
        } catch (IOException e) {
            e.printStackTrace();
            throw e;
        } finally {
            br.close();
        }

    }

    private static void printUntilNumber(int count) throws IOException {
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        try {
            for (int i = 1; i <= count; i++) {
                bw.write(i + "\n");
            }
            bw.flush();
        } finally {
            bw.close();
        }
    }

}

File: moregorenine/baekjoon/step3_for/q5_2741/main.js
const fs = require('fs')
const stdin = (process.platform === 'linux'
        ? fs.readFileSync(0, "utf8")
        : `5`
)

const count = parseInt(stdin)

printUntilNumber(count)

function printUntilNumber(count) {
    let result = ""
    for (let i = 1; i <= count; i++) {
        result += (i + '\n')
    }
    console.log(result)
}
